index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <!-- Start: Header -->
+    <div class="wrapper">
+      <header class="header">
+        <a href="#">My Logo</a>
+        <i class="fa-solid fa-bars"></i>
+      </header>
+     <!-- End: Header -->
+
+     <!-- Start: Main-->
+      <main>
+        <h2>I’m Hamza Baig. Glad to see <br> you!</h2>
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+        <h4>LET’S CONNECT</h4>
+        <section class="social-media">
+          <a href="#"><img src="images/twitter icon.svg" alt="Twitter icon"></a>
+          <a href="#"><img src="images/linkedin icon.svg" alt="Linkedin icon"></a>
+          <a href="#"><img src="images/m icon.svg" alt="M icon"></a>
+          <a href="#"><img src="images/github icon.svg" alt="Github icon"></a>
+          <a href="#"><img src="images/peace icon.svg" alt="Peace icon"></a>
```

Comment:
- [ ]  The list of icons should  use a `ul` tag to group them, and `li` per icon; each icon has `a` link (a tag).
Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.